examples/api-samples/package.json

```diff
@@ -10,6 +10,12 @@
   "theiaExtensions": [
     {
       "frontend": "lib/browser/api-samples-frontend-module"
+    },
+    {
+      "electronMain": "lib/electron-main/update/sample-updater-main-module"
+    },
+    {
+      "frontendElectron": "lib/electron-browser/updater/sample-updater-frontend-module"
```

Comment:
It should belong to the previous entry point?

Reply:
Yes, true. Thanks!